src/main/resources/artifact-ignores.properties

```diff
@@ -118,3 +118,16 @@ scm-api-2.0
 # Suppress Scala Hello World Plugin (HOSTING-235)
 hello-world-scala
 
+
+# Suppress potentially harmful upgrades related to SCM API (JENKINS-41121)
+# Erring on the side of extreme caution until the fallout is resolved here
+scm-api-2.0.0
+scm-api-2.0.1
+branch-api-2.0.0
```

Comment:
@jglick says we should also block workflow-multibranch 2.10  should be blocked as well